WordPress/Classes/System/WordPressAppDelegate.m

```diff
@@ -164,11 +164,8 @@ - (BOOL)application:(UIApplication *)application willFinishLaunchingWithOptions:
         [self setMustShowWhatsNewPopup:YES];
     }
     
-    CGRect bounds = [[UIScreen mainScreen] bounds];
-    [self.window setFrame:bounds];
-    [self.window setBounds:bounds]; // for good measure.
     self.window.rootViewController = [WPTabBarController sharedInstance];
-
+    [self.window makeKeyAndVisible];
```

Comment:
We're already calling this later in `application:didFinishLaunchingWithOptions:`